distributed/comm/tcp.py

```diff
@@ -552,5 +586,27 @@ class TLSBackend(BaseTCPBackend):
     _listener_class = TLSListener
 
 
+def handshake_info():
+    return {
+        "compression": default_compression,
+        "python": tuple(sys.version_info)[:3],
+    }
+
+
+def handshake_configuration(local, remote):
+    out = {}
+    if local["python"] >= (3, 8) and remote["python"] >= (3, 8):
+        out["pickle_protocol"] = 5
```

Comment:
Would we be able to check whether `pickle5` is available?

Reply:
Yeah, I was curious about how best to handle `pickle5`.  Good idea.  Done.